### PR TITLE
Fix popover style

### PR DESCRIPTION
**Pull request by emord**
@biyeun 

http://manage.dimagi.com/default.asp?157448#875043

This may be the wrong place to fix this. (not sure how you generate the bootstrap less files from the other bootstrap repo)

Right now popovers content will inherit whatever properties its parent element has (like h3) whereas popover titles overwrite these things (https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/static/style/lib/bootstrap/less/popovers.less#L37-L39).

Before pic is in ticket
After:
tooltip_007

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by orangejenny** [comment]
Can this be in a separate file - bootstrap_overrides.less or something? Not sure if there's a precedent for this, but directly editing bootstrap files is going to make upgrading a headache.

</unsolicited feedback>

File: corehq/apps/style/static/style/lib/bootstrap/less/popovers.less [hunk 1]
```diff
@@ -44,6 +44,9 @@
 
 .popover-content {
   padding: 9px 14px;
+  font-size: 12px;
```
Comment (biyeun) [inline corehq/apps/style/static/style/lib/bootstrap/less/popovers.less]: should be `@baseFontSize`

File: corehq/apps/style/static/style/lib/bootstrap/less/popovers.less [hunk 2]
```diff
@@ -44,6 +44,9 @@
...
+  font-weight: normal;
+  line-height: normal;
```
Comment (biyeun) [inline corehq/apps/style/static/style/lib/bootstrap/less/popovers.less]: should be `@baseLineHeight`

**Comment by biyeun** [comment]
@orangejenny this is that file, just without a redundant `_overrides`. @emord made the edits in the correct place...just need to reference the correct variables from less. Thanks for doing this! <3

**Comment by biyeun** [comment]
to clarify: everything in `style/lib/bootstrap` is an override (and extension, in some cases) of the correspondingly named file in the submodule.

**Comment by orangejenny** [comment]
Cool. Was confused by the similarity of this file and https://github.com/twbs/bootstrap/blob/master/less/popovers.less

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Pull request merged**